File: pset6/mario/mario.py
from cs50 import get_int

# request input from user
while True:
    n = get_int('Height: ')
    if n > 0 and n <= 8:
        break

# print the results
for i in range(n):
    print(' '*(n-i-1), end="")
    print('#'*(i+1))

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do 
    { 
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    for (int i = 1; i <= height; i++)
    {   
        for (int k = height - i; k > 0; k--)
        {
            printf(" ");
        }
        
        for (int j = 1; j <= i; j++)
        {    
            printf("#");
        }
        printf("\n");
    }

}
